zipkin/Dockerfile

```diff
@@ -11,20 +11,16 @@
 # or implied. See the License for the specific language governing permissions and limitations under
 # the License.
 #
-FROM openzipkin/jre-full:1.8.0_191
-MAINTAINER OpenZipkin "http://zipkin.io/"
+
+FROM alpine
 
 ENV ZIPKIN_REPO https://jcenter.bintray.com
 ENV ZIPKIN_VERSION 2.12.3
 
-# Use to set heap, trust store or other system properties.
-ENV JAVA_OPTS -Djava.security.egd=file:/dev/./urandom
-
-# Add environment settings for supported storage types
 COPY zipkin/ /zipkin/
```

Comment:
this looks repeated as also below

Reply:
Restored the comment - they look suspiciously the same but this copies from the repo (including the _profile) files, while the below one copies from this builder container (hence it has `--from=0`).

Reply:
this doesn't work for me using docker 18.09.2

Reply:
added a fix